### ISSUE TITLE
[Feature Request] Show Media Size before opening content

### ISSUE THREAD

**Issue by subpub** [ccrama/Slide#3074]
It would be helpful if media size is shown on the post cards before having to start loading them. Many times gifs and videos are too large than one would want to download, especially on mobile data.

**Comment by nikolowry** [comment]
This can be achieved with a HEAD request to the media right? I just wasted 80mb of data on two videos -- would love to see this feature implemented

**Comment by Alexendoo** [comment]
Yeah HEAD would work for that for most resources, it can also be done by reading the response headers of the normal request without download the body eagerly. Perhaps more difficult in wrangle into Slide but it wouldn't incur an extra round trip delay of loading

**Comment by nikolowry** [comment]
@Alexendoo it would also be fantastic if this could integrate into Data Saving -- possibly an alert dialog if media is over a certain MB

**Comment by gabcoroba** [comment]
Has this issue been resolved yet? I would like to help

**Comment by TacoTheDank** [comment]
@gabcoroba As far as I'm aware, not yet.

**Comment by hasnainmakada-99** [comment]
Hey I would like to work on implementing this feature of showing file size of media, before downloading it, Can you please tell me in which package & Which file it is ?
